modules/apps/foundation/frontend-theme/frontend-theme-user-dashboard/src/templates/portal_normal.ftl

```diff
@@ -52,15 +52,11 @@
 					</#if>
 				</div>
 
-				<#assign VOID = freeMarkerPortletPreferences.setValue("portletSetupPortletDecoratorId", "barebone") />
```

Comment:
Hey guys!

After a deeper look I think we can go ahead with this.
Right now user_personal_bar has not portlet container border, so we cannot configure its portlet decorator.

Thanks a lot!